dev-ruby/websocket/websocket-1.1.1.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+USE_RUBY="ruby20 ruby21"
+RUBY_FAKEGEM_RECIPE_DOC="none"
```

Comment:
You could use RUBY_FAKEGEM_RECIPE_DOC="rdoc" here to generate documentation using a generic approach that does not depend on the gem.